python/src/opendp/mod.py

```diff
@@ -258,6 +260,126 @@ def __iter__(self):
         # this overrides the implementation of __iter__ on POINTER, 
         # which yields infinitely on zero-sized types
         raise ValueError("Measurement does not support iteration")  # pragma: no cover
+    
+
+class Odometer(ctypes.POINTER(AnyOdometer)): # type: ignore[misc]
+    """A differentially private unit of computation with no privacy limit.
+    An odometer contains a function that returns a queryable with a function and a relation.
+    Differentially private queries may be passed to the queryable,
+    as well as queries to check the current privacy usage.
```

Comment:
```suggestion
    and ``privacy_loss()`` can be called to check the current privacy usage.
```
Calling them both "query" might suggest they have a similar interface.